ros2interface/ros2interface/verb/list.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ros2interface.api import get_all_action_types
+from ros2interface.api import get_all_message_types
+from ros2interface.api import get_all_service_types
+from ros2interface.verb import VerbExtension
+
+
+def printMessages():
+    print('Messages: ')
+    message_types = get_all_message_types()
+    for package_name in sorted(message_types.keys()):
```

Comment:
I don't think `.key()` is necessary.

Reply:
Why would it not be? Since message_types is a dict with the keys being the package names

Reply:
Be default, a for-in loop will iterate overs the keys of a dict.